src/Sql/Sql.Test/ScenarioTests/VulnerabilityAssessmentTests.ps1

```diff
@@ -35,11 +35,6 @@ function Test-VulnerabilityAssessmentServerSettingsTest
 		# Turn on ATP
 		Enable-AzSqlServerAdvancedThreatProtection -ResourceGroupName $params.rgname -ServerName $params.serverName -DoNotConfigureVulnerabilityAssessment
 
-		# Set Vulnerability Assessment settings (SetSettingsWithStorageAccountNameParameterSet parameter set) with a client errors
-		Assert-ThrowsContains -script { Update-AzSqlServerVulnerabilityAssessmentSettings -ResourceGroupName $params.rgname -ServerName $params.serverName `
-		 -StorageAccountName $params.storageAccount -EmailAdmins $false -NotificationEmail @() -RecurringScansInterval Weekly } `
-		 -message "You need to provide at least one email address or set EmailAdmins to True."
```

Comment:
need to remove the string "You need to provide at least one email address or set EmailAdmins to True" from the resource file

Reply:
or is TD uses it?

Reply:
TD is using it